### PR TITLE
Fix invalid license link

### PR DESCRIPTION
**Pull request by osallou**
Explicit set of license and add correct link to license file on github

* [ X ] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [ ] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ X ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/vphaser2/meta.yaml [hunk 1]
```diff
@@ -22,3 +23,8 @@ requirements:
 test:
     commands:
         - 'vphaser2 | grep "BAM file"'
+extra:
```
Comment (bgruening) [inline recipes/vphaser2/meta.yaml]: I think it is better to use this URL: https://github.com/broadinstitute/v-phaser2/archive/6b5483a8e1a9bf24061a0aa211fee18e6b845668.tar.gz and keep the linter.

File: recipes/vphaser2/meta.yaml [hunk 2]
```diff
@@ -1,6 +1,7 @@
 about:
     home: 'https://www.broadinstitute.org/scientific-community/science/projects/viral-genomics/v-phaser-2'
-    license: 'https://www.broadinstitute.org/node/4962'
+    license: 'single user license for academic non-commercial research purposes only'
+    license_file: 'https://github.com/broadinstitute/v-phaser2/blob/master/V-Phaser-2.0/license/V_Phaser_2_License_v20130317.pdf'
```
Comment (bgruening) [inline recipes/vphaser2/meta.yaml]: This should point to a file in the tarball.

Comment (osallou) [reply recipes/vphaser2/meta.yaml]: ok, I saw several other packages using references to github, so did the same, gonna fix this

File: recipes/vphaser2/meta.yaml (hunk 1 above)
Comment (osallou) [reply recipes/vphaser2/meta.yaml]: well, I only updated license part, I did not modify build part, but indeed refering to an archive would be better for reproducibility. I gonna update recipe to do so

Comment (osallou) [reply recipes/vphaser2/meta.yaml]: added archive instead of git_url, removing linter excludes

**Comment by bgruening** [comment]
This probably need a build-version bump to get a new release with the latest source.

File: recipes/vphaser2/meta.yaml [hunk 3]
```diff
@@ -1,14 +1,17 @@
...
-    license: 'https://www.broadinstitute.org/node/4962'
+    license: 'single user license for academic non-commercial research purposes only'
+    license_file: 'V-Phaser-2.0/license/V_Phaser_2_License_v20130317.pdf'
     summary: 'V-Phaser 2 is a tool to call variants in genetically heterogeneous populations from ultra-deep sequence data'
 package:
   name: "vphaser2"
   version: '2.0'
 source:
-  git_url: https://github.com/broadinstitute/v-phaser2.git #[osx or linux64]
+  url: https://github.com/broadinstitute/v-phaser2/archive/6b5483a8e1a9bf24061a0aa211fee18e6b845668.tar.gz  #[osx or linux64]
```
Comment (epruesse) [inline recipes/vphaser2/meta.yaml]: @bgruening Should this perhaps use cargo-port instead?

Comment (bgruening) [reply recipes/vphaser2/meta.yaml]: Lets put is this way, we do backup nowadays everything on cargo-port as soon as we merge it. To prevent data loss.

**Pull request merged**